src/snowflake/snowpark/dataframe.py

```diff
@@ -4330,43 +4330,38 @@ def convert(col: ColumnOrName) -> Expression:
         exprs = [convert(col) for col in parse_positional_args_to_list(*cols)]
         return exprs
 
-    def print_schema(self) -> None:
-        """
-        Prints the schema of a dataframe in tree format.
-
-        Examples::
-            >>> df = session.create_dataframe([(1, "a"), (2, "b")], schema=["a", "b"])
-            >>> df.print_schema()
-            root
-             |-- "A": LongType() (nullable = False)
-             |-- "B": StringType() (nullable = False)
-        """
+    def _format_schema(
+        self, level: Optional[int] = None, translate_columns: Optional[dict] = None
+    ) -> str:
```

Comment:
is the dict of translate_columns being passed by SAS project?

Reply:
I believe so. I've been talking with @sfc-gh-wshangguan about what they need from the api.